tests/gtest/avifclaptest.cc

```diff
@@ -65,6 +65,15 @@ constexpr InvalidClapPropertyParam kInvalidClapPropertyTestParams[] = {
      722,
      AVIF_PIXEL_FORMAT_YUV420,
      {330, 1, 385, 1, static_cast<uint32_t>(-308), 1, 103, 1}},
+    // pcX = -1/2 + (99 - 1)/2 = 48.5
+    // pcY = -1/2 + (99 - 1)/2 = 48.5
+    // leftmost = 48.5 - (99 - 1)/2 = -0.5 (not an integer)
+    // topmost = 48.5 - (99 - 1)/2 = -0.5 (not an integer)
+    {99,
+     99,
+     AVIF_PIXEL_FORMAT_YUV420,
+     {99, 1, 99, 1, static_cast<uint32_t>(-1), 2, static_cast<uint32_t>(-1),
```

Comment:
This is fine for consistency but please consider `uint32_t{-1}`.

Also the topic was already discussed but this unsigned-to-signed API should be reworked in my opinion, especially since the specification was updated. Is switching from `unsigned` to `signed` for some fields in `avifCleanApertureBox` a breaking API/ABI change? We should at least add a comment nearby this struct in avif.h to explain.

Reply:
This is the compiler warning for `uint32_t{-1}`:

```
libavif/tests/gtest/avifclaptest.cc:75:32: error: narrowing conversion of ‘-1’ from ‘int’ to ‘uint32_t’ {aka ‘unsigned int’} [-Wnarrowing]
   75 |      {99, 1, 99, 1, uint32_t{-1}, 2, static_cast<uint32_t>(-1),
      |                                ^
```

Reply:
> Also the topic was already discussed but this unsigned-to-signed API should be reworked in my opinion, especially since the specification was updated.

When you said "the specification was updated", were you referring to ISO BMFF or MIAF?

The eight fields in the clean aperture property still have the `unsigned int(32)` type in the latest versions of ISO BMFF and MIAF.

> We should at least add a comment nearby this struct in avif.h to explain.

Please see https://github.com/AOMediaCodec/libavif/pull/1749.

Reply:
> The eight fields in the clean aperture property still have the `unsigned int(32)` type in the latest versions of ISO BMFF and MIAF.

Indeed, I was mistaken. The version with `signed horizOffN` and `signed vertOffN` I was looking at is only a draft. Maybe it will get published some time soon.

> Please see #1749.

Thanks.